Update training-external-resources.md (#258)
bcgov/platform-developer-docs: src/docs/training-and-learning/training-external-resources.md

[The Twelve-Factor App](https://12factor.net/) - by Heroku (Adam Wiggins)

> [Microsoft Enterprise Skills Initiative](https://esi.microsoft.com/) by Microsoft. Free for BC Government employees through the contract between The Province and Microsoft. Useful for training and then proceeding to write certification exams to get Azure certified, 50% off exam costs as well.

Reviewer: ```suggestion
[Microsoft Enterprise Skills Initiative](https://esi.microsoft.com/) - by Microsoft. Free for BC Government employees through the contract between The Province and Microsoft. Useful for training and then proceeding to write certification exams to get Azure certified, 50% off exam costs as well. The Microsoft Azure Fundamentals course contains some broad cloud fundamentals that apply to OpenShift and other cloud technologies. 
```

> [Microsoft Enterprise Skills Initiative](https://esi.microsoft.com/) - by Microsoft. Free for BC Government employees through the contract between The Province and Microsoft. Useful for training and then proceeding to write certification exams to get Azure certified, 50% off exam costs as well. The Microsoft Azure Fundamentals course contains some broad cloud fundamentals that apply to OpenShift and other cloud technologies. 

Reviewer 2: ```suggestion
[Microsoft Enterprise Skills Initiative](https://esi.microsoft.com/) - by Microsoft. Free for B.C. government employees. Useful for training and take certification exams to get Azure certification. The exam has a 50% off discount. The Microsoft Azure Fundamentals course contains broad cloud fundamentals that apply to OpenShift and other cloud technologies. 
```

Reviewer: ```suggestion
[Microsoft Enterprise Skills Initiative](https://esi.microsoft.com/) - by Microsoft. Free for B.C. government employees through the contract between The Province and Microsoft. Useful for training and certification exams. The exam has a 50% off discount. The Microsoft Azure Fundamentals course contains some broad cloud fundamentals that apply to OpenShift and other cloud technologies. 
```
